### PR TITLE
Backend Testing

### PR DESCRIPTION
**Pull request by buk0vec**
- Created tests for User model
- Created tests for Page model
- Moved Express router into its own file
- Fixed ObjectID equality check issues
- Fixed some Typescript issues/assumptions in Page service
- Updated README with test coverage
- Added test cases to CI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,34 @@ For the backend server, a `.env` file is required to be in the `packages/sever`
...
+Output from running jest for server, 11/27/22 7:22 pm:
+
+```
+$ jest
+ PASS  src/user-service.test.ts (5.193 s)
+ PASS  src/page-service.test.ts (5.993 s)
+------------------|---------|----------|---------|---------|-------------------
+File              | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s
+------------------|---------|----------|---------|---------|-------------------
+All files         |     100 |      100 |     100 |     100 |
```
Comment (PeteThePilot) [inline README.md:28]: 100% wooooo

**Comment by PeteThePilot** [comment]
backend looking good

**Review by arthurz12345 — COMMENTED** [review]
Looks good to me

**Review by nichkhtan — APPROVED** [review]
Looks great!

**Pull request merged**